packages/flutter_tools/test/src/context.dart

```diff
@@ -234,3 +236,5 @@ class _MockUsageTimer implements UsageTimer {
 }
 
 class MockFlutterVersion extends Mock implements FlutterVersion {}
+
+class MockClock extends Mock implements Clock {}
```

Comment:
I think it's ok for the test to be verbose wrt the input/output near the assertions.